doc/tls/README.md

```diff
@@ -0,0 +1,311 @@
+# Issue the public site's certificate
+openssl x509 -req -days 100 \
+    -copy_extensions copyall \
+    -CA ca.crt -CAkey ca.key \
+    -in site-public.csr -out site-public.crt
+```
+
+This should produce the following files:
+
+`site-public.key`: The private key for the public site
+`site-public.crt`: The public key of the public site
+
+Add these new credentials to the public site.
+
+```
+kubectl create secret generic public-server-tls \
+    --from-file=ca.crt=ca.crt \
+    --from-file=tls.crt=site-public.crt \
+    --from-file=tls.key=site-public.key
+```
+Now we can issue a key pair for the private site to use for client authentication with the public site.
+
+```
+# Generate a private key
+openssl genrsa -out site-private.key 4096
+# Create a signing request for the private key
+openssl req -new \
+    -config network-ca.conf -section site \
+    -key site-private.key -out site-private.csr
+# Issue the private site's certificate
+openssl x509 -req -days 100 \
+    -copy_extensions copyall \
+    -CA ca.crt -CAkey ca.key \
+    -in site-private.csr -out site-private.crt
+```
+
+This should produce the following files:
+
+`site-private.key`: The private key for the private site
+`site-private.crt`: The public key of the private site
+
+Add these new credentials to the private site.
+```
+kubectl create secret generic public-link-tls \
+    --from-file=ca.crt=ca.crt \
+    --from-file=tls.crt=site-private.crt \
+    --from-file=tls.key=site-private.key
+```
+
+#### Link Sites
+
+At this point the public site should have a working RouterAccess and the
+private site should have TLS credentials that will allow it to authenticate.
+The private site needs a Link resource to complete the connection.
+
+The Link document needs to be populated with `tlsCredentials: public-link-tls`
+to match the name of the secret in the private site to use. It also requires a
+`endpoints` block, identical to the one in the public site's RouterAccess
+status section. The skupper CLI can generate this document for you, using the
+public kube context.
+
+```
+skupper link generate \
+    --tls-credentials public-link-tls \
+    --generate-credential=false | tee link.yaml
+```
+
+This will output a Link record document with the endpoints filled to connect to
+the public site.
+
+Using the kube context for the private site: `kubectl apply -f link.yaml`
+
+This will apply the link to the private site. The sites should connect.
+
+## TLS Credential Rotation
+
+**TODO** Support for rotating TLS Credentials is currently under development
+and is not yet fully implemented.
+
+When considering rotating the TLS credentials used by Skupper, it is important
+to understand the trust model used for linking Skupper Sites. The default
+[trust model](#controller-managed-tls-in-kubernetes) used by Skupper is
+distributed: each Site having its own trust root. Because of this, rotating
+client certificates and Site CAs is an especially complicated exercise.
```

Comment:
As a workaround, can I just recreate links and all connectors/listeners will 'just work'?